certificate/cert.crt

```diff
@@ -0,0 +1,35 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
TODO: These certs exist in the filesystem, so during development this all works fine. But when the service binaries are downloaded, they won't exist.

Strategy: probably `go:embed` them in the binary.. then spit them out into temp dirs?